src/scaffoldmaker/meshtypes/meshtype_3d_heartventricles2.py

```diff
@@ -749,10 +749,7 @@ def generateBaseMesh(cls, region, options):
 
                 element = mesh.createElement(elementIdentifier, elementtemplate1)
                 result2 = element.setNodesByIdentifier(eft1, nids)
-                if eft1.getNumberOfLocalScaleFactors() > 0:
-                    result3 = element.setScaleFactors(eft1, scalefactors)
-                else:
-                    result3 = 7
+                result3 = element.setScaleFactors(eft1, scalefactors) if scalefactors else None
```

Comment:
Is this supposed to be: result3 = element.setScaleFactors(eft1, scalefactors) if eft1.getNumberOfLocalScaleFactors() > 0 else None?

Reply:
I noticed that scalefactors is set to None at the start of the loop so it's more efficient and cleaner to check if it's True (i.e. not None) than call the eft function. Plus it matches the other code. 2 places allowed this change.

Reply:
ok sounds good